doc/schema.json

```diff
@@ -1,5 +1,5 @@
 {
-    "$schema": "http://json-schema.org/draft-04/schema#",
+    "$schema": "https://json-schema.org/draft-04/schema#",
```

Comment:
Looks like the certificate for this one is not valid, I get a "This connection is not secure" page when trying to reach it.